README.md

```diff
@@ -1,2 +1,83 @@
-# arduino-ide-esp-exception-decoder
-Arduino IDE ESP8266/ESP32 Exception Stack Trace Decoder Extension
+# ESP Exception Decoder
+
+[![Tests](https://github.com/dankeboy36/esp-exception-decoder/actions/workflows/ci.yml/badge.svg)](https://github.com/dankeboy36/esp-exception-decoder/actions/workflows/ci.yml)
+
+> ⚠️ This project is in an early state.
+
+[Arduino IDE](https://github.com/arduino/arduino-ide/) extension lets you get a more meaningful explanation of the stack traces and backtraces you get on ESP8266/ESP32. This extension is the reimplementation of the famous [ESP8266/ESP32 Exception Stack Trace Decoder](https://github.com/me-no-dev/EspExceptionDecoder) tool, written in Java.
+
+![ESP8266/ESP32 Exception Decoder Extension](./images/espExceptionDecoder_main.png)
+
+> ⚠️ This extension has nothing to do with the [Visual Studio Code extension for Arduino](https://marketplace.visualstudio.com/items?itemName=vsciot-vscode.vscode-arduino). This extension does not work in VS Code.
+
+## Installation
+
+- Download the latest extension from the GitHub [release page](https://github.com/dankeboy36/vscode-arduino-api/releases/latest). The asset filename is `esp-exception-decoder-${VERSION}.vsix`, where `${VERSION}` is the latest version.
```

Comment:
Incorrect link to the release page. It should be `esp-exception-decoder`.

Reply:
https://github.com/dankeboy36/esp-exception-decoder/pull/1/commits/7b31608263e5e304f3ac0adf4937756b5c3a28b7